Microverse Project #1: Embeding Images and Video.

SigmaSam opened the pull request:
Cleaned the code from comments.
Removed ineccesary code segments.
Restructured the general appearance of the page.

KennyUTN committed:
Pull Request Problems!

SigmaSam committed:
Fixed the position of the related videos.

Added views.

KennyUTN committed:
Final Corrections Push

serdg0 reviewed (changes requested):
**Status: Require Changes**
### Hello Ignacio and Samuel! :v: 
Great work on your first project! Hope you guys are enjoying your time here! :smiley: 
You are in the right path, but we require some changes for this project to be outstanding! 
Remember, this is going to be seen in the future by potential employers :moneybag: :moneybag: 
Here they are:

**Navbar**

- [x] Icons .- Let's add the corresponding icons from the original webpage. [Here](https://icons8.com/icons ) is a suggestion for icons.

![icons](https://user-images.githubusercontent.com/49207642/78341264-9ffbbf80-7554-11ea-812f-254924834624.png)

**Sidebar**

- [x] Positioning .- It does count wih the right elements but there is a big gutter between the video and the sidebar. Try to stick to the original webiste.

![sidebar](https://user-images.githubusercontent.com/49207642/78341471-02ed5680-7555-11ea-99f7-3c58e1a1a67c.png)

**Comments**

- [x] Scroll .- Let's get rid of the scroll. The comment section should be displayed in the whole rest of the page, as seen in the original website.

![scroll](https://user-images.githubusercontent.com/49207642/78341752-6e372880-7555-11ea-9bbf-a7a5c5180398.png)

**Body**
![margin](https://user-images.githubusercontent.com/49207642/78342022-de45ae80-7555-11ea-940b-61dc226a0eec.png)

- [x] Positioning. As seen in the picture, this element is surrounded by a margin, which makes other elements differ from their original positions (website)

Also, you don't need to re-open a new PR when sending for a code review, your commits will persist in the same PR :smile: 
Remember, this project is about mimicking as close as possible the original website.
This project was challenging for me when I did it! So don't hesitate to reach me in slack for any doubt you may have **@serdg** 


### Happy Coding! :computer: :computer:

Serdg commented:
Sorry, but @serdg0

SigmaSam committed:
Added Linters.

KennyUTN committed:
subscribe button

SigmaSam committed:
Playing with the grid positon and button size.

KennyUTN committed:
Update subscribe.png

Subscribe BUtton

KennyUTN committed:
Merge branch 'pullin' of https://github.com/SigmaSam/Embeding-Images-and-Video-Ignacio-y-Samuel. into pullin

KennyUTN committed:
Update subscribe.png

susbscribe button again

KennyUTN committed:
Updating format

SigmaSam committed:
Finished.

SigmaSam committed:
Changed the background.

SigmaSam committed:
Ajusted Item-B, Padding.

KennyUTN committed:
Added readme and Preview

SigmaSam commented:
Changed the navigation bar and video information icons.
Repositioned the elements to the same distance from the navigation bar.
Fixed the distance between the video and the sidebar.
Removed the comment section as it wasn't in the original website.

SigmaSam committed:
Updated Readme.

men32z commented on styleSheet.css:
```diff
@@ -0,0 +1,264 @@
+}
+
+body {
+  /* margin: 10px 10px; */
```
remove commented code please

men32z reviewed (changes requested):
Hey team!
## Good job on this project! 👍

There are still some changes 👇 you need to implement to approve this request 🔍
- [ ] seems that you remove all comments section, kindly add it again, the previous TSE ask to remove just the div scroll, you don't have to worry about the normal scroll of the browser.

- [ ] Files not related to the project should not be committed and added to .gitignore (e.g. `.DS_Store`) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/gitignore.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/)) (the .mp4 file for instance)
- [ ] Comments: It’s better to delete commented code. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))

## Please, make all REQUIRED changes and submit your request again! 😀

If you have any questions, feel free to send me a message through Slack (@men32z)
I would be happy to help! 👩‍💻

Happy coding 🤓

--Luis Preza

KennyUTN committed:
Fixing  required problems

KennyUTN committed:
Final Fixes

christianotieno commented:
## Pull Request under review by Christian Otieno (TSE)

christianotieno reviewed (changes requested):
# **CHANGES REQUIRED** :warning:
Hey Team :wave:
-
You have done a splendid job! :tada: Your team spirit is so infectious. Keep it up.

We are almost through with this. Let's [validate](https://validator.w3.org/nu/#textarea) our markup and remove all the highlighted errors before we get approved. Thank you.
##
Please makes changes and submit another review request.
- If you have any questions, let me know here or on slack and I will be glad to clarify with you. You can also reach me and connect with me on my social media or email. :eyes:
##
- When you have made the specified changes, please submit another code review. Thank you!
Happy coding :smile:
**[Christian Otieno](https://github.com/ChristianOtieno)**
[Twitter](https://twitter.com/iamchrisotieno) [linkedin](https://www.linkedin.com/in/christianotieno/) [GitLab](https://gitlab.com/christianotieno)
---
*P.S. all requested changes not marked by [OPTIONAL] or are not positive comments are **REQUIRED***

SigmaSam committed:
Validated Index.html and styleSheet.css.

SigmaSam commented:
Validated the HTML and CSS code as requested.

mikenath223 commented:
### Pull request currently under review by Michgolden Ukeje (TSE).
Kindly exercise patience 👋

mikenath223 reviewed (changes requested):
### Hi Team 👋👋

**WELCOME TO MICROVERSE.**👏

You've done quite well on this for a first project.
Great job working with previous reviewers on this.👍

However, there are a  few changes that need to be made before proceeding.

 - [ ] Required - Kindly ensure you are making your clone from this [link](http://archive.today/Bss88) as stated in the microverse adjustment in Partwright. See [screenshot]👇
![ice_screenshot_20200408-080224](https://user-images.githubusercontent.com/33323077/78755092-82719000-7970-11ea-950b-3d6a29899bcb.png)

-  [ ] Required - Also kindly validate your HTML code using this HTML5 validator [tool](https://validator.w3.org/nu/#file).

- [ ] Required - As seen in that link, kindly add a footer section to your page,  a `Choose your language` template section also and fix the necessary requirement as stated in the screencast below.

Please watch this screencast for more info👉[SCREENCAST](https://www.loom.com/share/7f4cf2bdac2a499e91295466990ee503)

Kindly fix these issues and request another review👍
Feel free to contact me on slack(michgolden) if you need further clarification.

HAPPY CODING👨‍💻

SigmaSam committed:
Added video description area.

SigmaSam committed:
Added footer.

SigmaSam committed:
Added a global grid.

KennyUTN committed:
Updated readme and preview

mikenath223 reviewed (changes requested):
### Hello again Team 👋

Well done on this @SigmaSam  and @KennyUTN 👍
Must have taken you a lot of hard work to get to this stage.

However, kindly exercise patience, as we still got a few more changes to make and you will be all set.💪

Please note that this project involves making a clone(copy) of the original website and as such there should be little or no difference between it and the original [website](http://archive.vn/Bss88).
Kindly make these necessary adjustments:👇

- [ ] Required - Check HTML markup with this [validator](https://validator.w3.org/#validate_by_upload). ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/validation_requirement.png) from Pathwright) 

- [ ] Required - Please add a `Choose your language` section just below the header of your webpage.[Screenshot]👇
![ice_screenshot_20200409-051226](https://user-images.githubusercontent.com/33323077/78883178-08650800-7a51-11ea-99f2-74dcf99ff9de.png)

- [ ]  Required - In addition kindly make adjustments to the section just below your video element. Specific items need to be grouped together.[Screenshot]
**Kindly group these items together:**
This is yours👇
![ice_screenshot_20200409-103112](https://user-images.githubusercontent.com/33323077/78883484-790c2480-7a51-11ea-8a95-b9789f582d50.png)

This is the way it should be👇
![ice_screenshot_20200409-103206](https://user-images.githubusercontent.com/33323077/78883529-87f2d700-7a51-11ea-89d6-bc91b16bd8b4.png)

* Also add a video title above the author avatar as seen in the original👆

**Group these items together too.**
This is yours:
![ice_screenshot_20200409-103125](https://user-images.githubusercontent.com/33323077/78883766-da33f800-7a51-11ea-887f-89199c96693e.png)

This is the way it should be:
![ice_screenshot_20200409-103218](https://user-images.githubusercontent.com/33323077/78883898-051e4c00-7a52-11ea-9874-ec4385f41bdd.png)

- [ ] Required - Add an `Upload` icon and text to the top of the sidebar video section.👇
![ice_screenshot_20200409-103444](https://user-images.githubusercontent.com/33323077/78886164-ace94900-7a55-11ea-8de4-03f0c945fd35.png)


- [ ] Required - Great job adding a footer section, however, kindly add this links that come below it.[Screenshot]👇
![ice_screenshot_20200409-103351](https://user-images.githubusercontent.com/33323077/78885999-70b5e880-7a55-11ea-9dfc-1d602d8977fd.png)

If you make these changes following each point carefully, you will surely ace the next review.💪👏🤗

Kindly fix these issues and request another review👍
Feel free to contact me on slack(michgolden) if you need further clarification.

HAPPY CODING👨‍💻

SigmaSam committed:
Added recomendations.

SigmaSam committed:
Validated the code and cleaned the errors.

SigmaSam committed:
Deleted unnecesary media.

christianotieno reviewed (approved):
# PROJECT APPROVED
***All the minimum requirements have been met in this project. I will go ahead and mark this as complete.***
- Congratulations team! :+1:
- You can now merge to master and I wish you all the very best in your next tasks :first_place_medal:
##
Happy coding :smile:
**[Christian Otieno](https://github.com/ChristianOtieno)**
[Twitter](https://twitter.com/iamchrisotieno) [linkedin](https://www.linkedin.com/in/christianotieno/) [GitLab](https://gitlab.com/christianotieno)
